### PR TITLE
feat(*): minor polish of /accountcreated page

### PR DESCRIPTION
**Pull request by piperchester**
Ran into this during testing #674. Very minor polish and opinionated wording of `/accountcreated`. I think this is a small improvement, but happy to edit, or back out any changes.

## Before
Screen Shot 2021-08-17 at 6 02 08 PM

## After
Screen Shot 2021-08-17 at 6 01 54 PM

### CODE REVIEW

File: frontend/public/texts/profile_texts.js [hunk 1]
```diff
@@ -166,23 +174,27 @@ export default function getProfileTexts({ profile, locale }) {
...
+      en: "Please click on the link we just emailed you to activate your account.",
+      de: "Bitte klicke auf den Link, um dein Konto zu aktivieren.",
+    },
     please_click_on_the_link_to_activate_your_account: {
       en: "Please click on the link to activate your account.",
       de: "Bitte klicke auf den Link, um dein Konto zu aktivieren.",
     },
     make_sure_to_also_check_your_spam: {
-      en: "Make sure to also check your spam/junk folder incase you cannot find the E-Mail.",
+      en: "Make sure to also check your spam/junk folder in case you can't find the link.",
```
Comment (piperchester) [inline frontend/public/texts/profile_texts.js:186]: @positiveimpact I'll need help on the 🇩🇪

**Review by ddhanesha — APPROVED** [review]
Lets get this merged

**Comment by piperchester** [comment]
> Lets get this merged

Just need help on the German from Chris and then I think we're good!

**Review by positiveimpact — COMMENTED** [review]
Looks good to me overall. The german all looks pretty good! 
The only thing left to do would be to remove the unused texts from the text files

File: frontend/pages/accountcreated.js [hunk 2]
```diff
@@ -40,21 +40,22 @@ export default function AccountCreated() {
     return (
       <Layout title={texts.account_created} hideHeadline>
         <Paper className={classes.root}>
-          <Typography variant="h5" className={classes.headline}>
-            {texts.congratulations_just_one_more_step_to_complete_your_signup}
```
[the remark this answers is not in the document]
Comment (piperchester) [reply frontend/pages/accountcreated.js:44]: Yeah great call! I'll do that now.

[the remark this answers is not in the document]
Comment (piperchester) [reply frontend/pages/accountcreated.js:44]: Ok deleted in 490a1f1 and 1b13ebd.

**Pull request merged**